<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditGroupWidgetKeeShare</class>
 <widget class="QWidget" name="EditGroupWidgetKeeShare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="MessageWidget" name="messageWidget" native="true"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="typeComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="passwordLayout">
       <item>
        <widget class="PasswordEdit" name="passwordEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="togglePasswordButton">
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="togglePasswordGeneratorButton">
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="PasswordGeneratorWidget" name="passwordGenerator" native="true"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="pathLabel">
       <property name="text">
        <string>Paths:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="PathLineEdit" name="pathLineEdit" native="true"/>
     </item>
     <item row="4" column="1">
      <widget class="QWidget" name="pathOverrides" native="true">
       <layout class="QVBoxLayout" name="pathOverridesLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="pathLocalOverrides" native="true">
          <layout class="QFormLayout" name="pathLocalOverridesLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pathLocalPreview">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&lt;preview&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="pathRemoteOverrides" native="true">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <layout class="QFormLayout" name="pathRemoteOverridesLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PasswordGeneratorWidget</class>
   <extends>QWidget</extends>
   <header>gui/PasswordGeneratorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PasswordEdit</class>
   <extends>QLineEdit</extends>
   <header>gui/PasswordEdit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MessageWidget</class>
   <extends>QWidget</extends>
   <header>gui/MessageWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PathLineEdit</class>
   <extends>QWidget</extends>
   <header>keeshare/PathLineEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
